crates/matrix-sdk-crypto/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # UNRELEASED
 
+Changed:
+
+- Fallback keys are rotated in a time-based manner, instead of waiting for the
```

Comment:
Is it a breaking change? regarding the migration part?

Reply:
No, no migration code needed, the pickle gets a new `Option` field that by default remains `None`. Though that does mean that we're going to rotate the fallback key when we migrate and convert the `None` into a `Some`.